coreneuron/sim/fadvance_core.cpp

```diff
@@ -30,6 +30,45 @@ extern corenrn_parameters corenrn_param;
 static void* nrn_fixed_step_thread(NrnThread*);
 static void* nrn_fixed_step_group_thread(NrnThread*, int, int, int&);
 
+
+namespace {
+
+class ProgressBar {
+    bool show;
```

Comment:
Nitpick: better to put this after `pbar` and `current_step` to keep `sizeof(ProgressBar)` smaller.

Reply:
the lost art of struct packing :)